api/configure.md

```diff
@@ -0,0 +1,6 @@
+---
+uid: configure
+title: Configuration Operators
+---
+
+Configuration operators belong in a top-level chain of operators between [core operators](xref:core) (`CreateContext` and `StartAcquisition`) to configure ONIX hardware.
```

Comment:
`CreateContext` and `StartAcquisition` should be cross-referenced so users can click a link.

```
[`CreateContext`](xref:OpenEphys.Onix1.CreateContext)
```